controller/config.go

```diff
@@ -375,6 +391,20 @@ func Validate(c Config) error {
 		}
 	}
 
+	if v, ok := c[JujuHASpace].(string); ok {
+		if !names.IsValidSpace(v) {
+			return errors.NewNotValid(nil, "invalid juju HA space name")
+		}
+
```

Comment:
no need for the extra blank spaces here, though shouldn't we do something if !ok ?
specifically, I would have expected an
} else {
  // !ok
  return errors.NewNotValid(nil, "juju-ha-space must be a string")